istioctl/cmd/proxyconfig_test.go

```diff
@@ -76,28 +77,33 @@ func TestProxyConfig(t *testing.T) {
 			expectedString: "unable to retrieve Pod: pods \"invalid\" not found",
 			wantException:  true, // "istioctl proxy-config routes invalid" should fail
 		},
-		{ // case 5 bootstrap invalid
+		{ // case 6 bootstrap invalid
```

Comment:
i don't understand why proxyconfig_test.go need to document this "case 1,2,3..." can't see how it helps with readability.

Also seems like case 5 now is missing?

Reply:
agreed, also tough to add new tests because you have to shift all the numbers up. I'll remove the numbering if that's fine with @esnible

Reply:
@Monkeyanator You may get rid of the numbering.